### PR TITLE
Git templates

### PR DESCRIPTION
**Pull request by DivyaVavili**
Adding templates for Issue and PR creation

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+#### Expected Behavior
+<!--- Describe the expected behavior for the scenario/use case -->
+
+#### Observed Behavior
+<!--- If describing a bug, tell us what happened instead of the expected behavior -->
+<!--- If suggesting a feature request/enhancement, explain the mprovement from the current behavior -->
+
+## Steps to Reproduce (for bugs)
+<!--- Please describe th steps to reproduce this bug -->
```
Comment (dseevr) [inline .github/ISSUE_TEMPLATE.md]: th -> the

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<!--- Thanks for opening an issue.
+
+If this is an issue, please fill in as much details as possible in the bug report information.
```
Comment (dseevr) [inline .github/ISSUE_TEMPLATE.md]: much -> many

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+If this is a feature request/enhancement please mention the use-case, expected behavior and examples as necessary.
```
Comment (dseevr) [inline .github/ISSUE_TEMPLATE.md]: missing comma after "enhancement"

no hyphen in "use case"

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+## Description
+<!--- If you're describing a bug, tell us what happened -->
+<!--- If you're suggesting a feature request/enhancement, tell us what you would like to see -->
+
+#### Expected Behavior
+<!--- Describe the expected behavior for the scenario/use case -->
+
+#### Observed Behavior
+<!--- If describing a bug, tell us what happened instead of the expected behavior -->
+<!--- If suggesting a feature request/enhancement, explain the mprovement from the current behavior -->
```
Comment (dseevr) [inline .github/ISSUE_TEMPLATE.md]: mprovement -> improvement

File: .github/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Steps to Reproduce (for bugs)
+<!--- Please describe th steps to reproduce this bug -->
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
```
Comment (dseevr) [inline .github/ISSUE_TEMPLATE.md]: "in which you experienced the bug"

**Comment by DivyaVavili** [comment]
Whoa. Needed a spell-checker. Guess this is what happens when using vim and doing it in a hurry.

**Pull request merged**